data/questions_gen_postgres.csv

```diff
@@ -160,7 +160,7 @@ What is the total number of papers published in each year?,"SELECT paper.year, C
 What is the total number of papers associated with each dataset?,"SELECT paperdataset.datasetid, COUNT(DISTINCT paperdataset.paperid) AS total_papers FROM paperdataset GROUP BY paperdataset.datasetid;SELECT dataset.datasetname, COUNT(paperdataset.paperid) AS total_papers FROM paperdataset JOIN dataset ON paperdataset.datasetid = dataset.datasetid GROUP BY dataset.datasetname;",scholar,group_by,
 How many keyphrases are associated with each paper?,"SELECT paperkeyphrase.paperid, COUNT(paperkeyphrase.keyphraseid) AS keyphrase_count FROM paperkeyphrase GROUP BY paperkeyphrase.paperid ORDER BY keyphrase_count DESC NULLS LAST;SELECT p.title, COUNT(pk.keyphraseid) AS num_keyphrases FROM paper p JOIN paperkeyphrase pk ON p.paperid = pk.paperid GROUP BY p.title ORDER BY num_keyphrases DESC NULLS LAST;",scholar,group_by,
 How many authors have published more than 2 papers?,SELECT COUNT(*) AS number_of_authors FROM (SELECT writes.authorid FROM writes GROUP BY writes.authorid HAVING COUNT(writes.paperid) > 2) AS subquery;,scholar,group_by,
-"Which papers have the highest number of authors, ordered by the number of authors in descending order?","SELECT writes.paperid, COUNT(writes.authorid) AS num_authors FROM writes GROUP BY writes.paperid ORDER BY num_authors DESC NULLS LAST;SELECT paper.title, COUNT(DISTINCT writes.authorid) AS num_authors FROM paper JOIN writes ON paper.paperid = writes.paperid GROUP BY paper.title ORDER BY num_authors DESC;",scholar,order_by,
+"Which papers have the highest number of authors, ordered by the number of authors in descending order?","SELECT {paper.paperid, paper.title}, COUNT(DISTINCT writes.authorid) AS num_authors FROM paper JOIN writes ON paper.paperid = writes.paperid GROUP BY {} ORDER BY num_authors DESC;",scholar,order_by,

```

Comment:
As there is a tie here in the top 3 papers, I'll add some more data to `defog-data` to ensure that we break the ties via the data since we only need to verify the ordering by number of authors and not the paper id/name:
```
 paperid |                    title                     | num_authors 
---------+----------------------------------------------+-------------
       1 | A Study on Machine Learning Algorithms       |           3
       2 | The Effects of Climate Change on Agriculture |           3
       4 | COVID-19 Impact on Society                   |           3
       5 | Machine Learning in Tackling Climate Change  |           2
       3 | Social Media and Mental Health               |           1
```

Reply:
Good point, though the eval logic now takes "tie-breaks" into account for order by questions

Reply:
New logic here! I _think_ we should be good to go without changes to defog-data

https://github.com/defog-ai/sql-eval/blob/b8b77041e44de3095b5e2d98f9ba103a16790390/eval/eval.py#L42